docs/arithmetization/tables/mem-before.tex

```diff
@@ -0,0 +1,14 @@
+\subsection{MemBefore}
+\label{mem-before}
+
+The MemBefore holds the content of the memory before the execution of the current segment. For consistency, it must be exactly the same as the MemAfter trace
+of the previous segment.
+Each row of the MemBefore table contains:
+
+\begin{enumerate}
+  \item $a$, the memory cell address,
+  \item $v$, the initial value of the cell.
+\end{enumerate}
+The MemBefore table should be ordered by $(a, \tau)$. Since it only holds values, there are no constraints between the rows.
+
+A CTL copies all of these initial values in the memory trace as reads, at timestamp $\tau = 0$.
```

Comment:
We can remove this section entirely, it's copy-pasta and we said above these two tables were literally identical.